multinode-demo/common.sh

```diff
@@ -28,9 +28,9 @@ fi
 if [[ -d $SNAP ]]; then # Running inside a Linux Snap?
   solana_program() {
     declare program="$1"
-    if [[ "$program" = wallet || "$program" = bench-tps ]]; then
-      # TODO: Merge wallet.sh/client.sh functionality into
-      #       solana-wallet/solana-demo-client proper and remove this special case
+    if [[ "$program" = bench-tps ]]; then
+      # TODO: Merge client.sh functionality into
```

Comment:
I think this comment can die, now.  The script is invoking bench-tps directly, not via client.sh.

Reply:
Am I correct that the whole `if` statement should die? Or just the comment?

Reply:
🍄 the whole `if` please

Reply:
I'm not feeling so sure, now

Let me look at what command-%s.wrapper is...

Reply:
um, can't find it...  I think that means the clause should die?